### ISSUE TITLE
How to get Entity after put operation

### ISSUE THREAD

**Issue by BraunreutherA** [pushtorefresh/storio#623]
Hi there,

I'm inserting entities into the database and get a putresult in my observable. Is there a way to get the entity that was inserted/updated aswell?

Thanks,
Alex

**Comment by artem-zinnatullin** [comment]
You want to retrieve it from `PutResult`?

**Comment by BraunreutherA** [comment]
Yes - or another way. But it would be nice to get the inserted object and not just if it was inserted

**Comment by artem-zinnatullin** [comment]
At the moment I'd suggest to simply retain the object that you're putting yourself, I think we'll add it to `PutResult` soon since `PutResults` (represents put of multiple objects) already stores them.

**Comment by BraunreutherA** [comment]
Thats the way I do it at the moment. Great when you can put it on the next drive :+1:

**Comment by Rainer-Lang** [comment]
@artem-zinnatullin @nikitin-da Could I get now the inserted/updated object from PutResult?

**Comment by nikitin-da** [comment]
Sorry, we still have not done this. I hope it will be available in next release
